app/controllers/webhook_deliveries_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WebhookDeliveriesController < ApplicationController
+  respond_to :html, :js
+
+  before_action :set_namespace
+  before_action :set_webhook
+  before_action :set_webhook_delivery
```

Comment:
But IMHO it's best practice to put that into an action. Already prepared if it gets more methods.